converter/prototype.py

```diff
@@ -153,7 +153,8 @@ def get_destination_settings_if_any(
 
             if "overlayBackgroundInteraction" in transition_node:
                 overlay_settings = FlowOverlaySettings.Positioned(
-                    transition_node.get("overlayPositionType", "CENTER")
+                    transition_node.get("overlayPositionType", "CENTER"),
+                    action.get("overlayRelativePosition"),
```

Comment:
Are the relative positions the same in fig and Sketch? This looks like the kind of thing that needs to be converted to the (0, 1) internal for Sketch to work, but maybe not.

If this works like this, great :sparkle: It's just a bit surprising but I cannot complain.

Reply:
I was surprised too. I tested it positioning it at different places, including going "off screen", and it always got converted right so 🤷‍♀️